samples/core/Benchmarks/AverageBlogRanking_20210107.txt

```diff
@@ -0,0 +1,57 @@
+// * Summary *
```

Comment:
If this file is not used in doc pages then should remove it. It represent one snapshot of benchmark run. User may get confused seeing different data when they run.

Reply:
Yeah, also note that there are also some results [inline on the page](https://docs.microsoft.com/en-us/ef/core/performance/performance-diagnosis?tabs=simple-logging%2Cload-entities#benchmarking-with-ef-core).